Dockerfile

```diff
@@ -1,93 +1,98 @@
-FROM ubuntu:18.04
-MAINTAINER Quartile Limited <info@quartile.co>
+FROM debian:buster-slim
```

Comment:
Wouldn't it be better to use `python:3.8-slim-buster`? Debian 10 comes with Python 3.7 which I reckon will be kind of "the least tested" version among the Odoo community.

Reply:
I tried to use `python:3.8-slim-buster` and compare the size which is 500 mb bigger. Therefore I choose to use `debian:buster-slim` in the end.

Reply:
What's your idea about the Python version, though? OCA will not be testing with Python 3.7, so I don't feel very safe with this version.